app.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    return "Main page for video game inclusion."
+
+
+@app.route('/<resource>', methods=['GET', 'POST'])
+def info_pull(resource):
+    assert resource == request.view_args['resource']
+    return "Pull general information on video games that are being requested."
+
+
+@app.route('/<resource>/uuid', methods=['GET', 'PATCH', 'DELETE'])
```

Comment:
```suggestion
@app.route('/videogames/<uuid>', methods=['GET', 'PATCH', 'DELETE'])
```

The UUID is the variable part of this url